.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -97,16 +41,16 @@ body:
     id: operating-system-version
     attributes:
       label: Which operating system version?
-      placeholder: e.g. Debian 11
+      placeholder: Debian 11 (Bullseye)
     validations:
       required: true
   - type: dropdown
     id: isa
     attributes:
-      label: What instruction set architecture is DMS running on?
+      label: What computer architecture is the host running?
       options:
-        - AMD64 / x86_64
-        - ARM64 / AArch64 (ARM v8 and newer)
+        - AMD64 (x86_64)
+        - ARM64 (ARMv8 and newer)
```

`AArch64` is equivalent to `x86_64` IIRC?  ARMv8 is the oldest supported, while ARMv7 is for `AArch32`: https://stackoverflow.com/questions/31851611/differences-between-arm64-and-aarch64

```suggestion
        - ARM64 (AArch64)
```